eos/form-factors/b-lcdas-flvd2022.cc

```diff
@@ -315,7 +320,7 @@ namespace eos
         FLvD2022::options
         {
             { "q",       { "u", "s" },           "u"        },
-            { "g-minus", { "zero", "WW-limit" }, "WW-limit" }
+            { "gminus", { "zero", "WW-limit" }, "WW-limit" }
```

Comment:
Is this compatibl with how the options are used in Nico's implementation of the light-cone sum rules?
E.g. ``eos/form-factors/analytic-b-to-p-lcsr-impl.hh``, I think.